src/content/index.mdx

```diff
@@ -2,8 +2,38 @@
 title: "Home"
 metaTitle: "Neural Magic Documentation"
 metaDescription: "Documentation for the Neural Magic Deep Sparse Platform a suite of software components to train and deploy sparsified deep learning models performantly on your data"
-githubURL: ""
+githubURL: "https://github.com/neuralmagic/docs/blob/main/src/content/index.mdx"
 index: 0
 ---
 
-# Home
+# Neural Magic Documentation
+
+Neural Magic’s Deep Sparse Platform provides a suite of software components to train and deploy sparsified deep learning models performantly on your data.
+The [DeepSparse Engine](/products/deepsparse) enables GPU class performance on CPUs for neural network deployments.
+[SparseML](/products/sparseml) leverages [recipes](/user-guide/what-are-recipes) to allow simple model [sparsification](/user-guide/what-is-sparsification) with only a few lines of code.
+And the [SparseZoo](/products/sparsezoo) stores presparsified models and sparsification recipes so you can easily plugin your data.
+
+<LinkCards>
+  <LinkCard href="/doc/get-started" heading="Get Started">
+    A step-by-step introduction into basic Deep Sparse Platform features
+  </LinkCard>
+
+  <LinkCard href="/doc/user-guide" heading="User Guide">
+    Study the detailed inner-workings of the Deep Sparse Platform in its user guide.
+  </LinkCard>
+
+  <LinkCard href="/doc/use-cases" heading="Use Cases">
+    Non-exhaustive list of scenarios the Deep Sparse Platform can help with
+  </LinkCard>
+
+  <LinkCard href="/doc/command-reference" heading="Command Reference">
+    See all of the Deep Sparse Platform's commands.
+  </LinkCard>
+</LinkCards>
+
+✅ Please join our [community](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ) if you need any help
+or [subscribe](https://neuralmagic.com/deep-sparse-community/#subscribe) for regular Neural Magic email updates. ⚡
+
+✅ Check out our [GitHub repositories](https://github.com/neuralmagic) and give us a ⭐ if you like the projects!
+
+✅ Contribute to DVC [on GitHub](https://github.com/neuralmagic) or help us improve this [documentation](https://github.com/neuralmagic/docs) 🙏.
```

Comment:
nit: Use dvc link instead of NM github
```suggestion
✅ Contribute to DVC [on GitHub](https://github.com/iterative/dvc.org/blob/main/content/docs/user-guide/what-is-dvc.md) or help us improve this [documentation](https://github.com/neuralmagic/docs) 🙏.
```